extension/src/popup/components/manageAssets/ManageAssetRowButton/index.tsx

```diff
@@ -130,12 +133,6 @@ export const ManageAssetRowButton = ({
       );
 
       if (submitFreighterTransaction.fulfilled.match(submitResp)) {
-        dispatch(
-          getAccountBalances({
-            publicKey,
-            networkDetails,
-          }),
-        );
```

Comment:
after we change a trustline, why do we not want to re-fetch balances? Our list of balances should be different after tx submission